councilmatic/settings_jurisdiction.py

```diff
@@ -4,10 +4,10 @@
 # These settings are required #
 ###############################
 
-OCD_CITY_COUNCIL_ID = 'ocd-organization/ef168607-9135-4177-ad8e-c1f7a4806c3a'
-CITY_COUNCIL_NAME = 'Chicago City Council'
+OCD_CITY_COUNCIL_ID = 'ocd-organization/3266e69a-85ed-4c76-a98f-5652081ab5d1'
```

Comment:
i'm still not sure why this change was necessary?

Reply:
i bet it's because it's referring to my local org, not the one in the api. oop.

Reply:
the id is created when the organization is created, i.e., there isn't a canonical id to reference here, because it will be created when we initialize the chicago-specific ocd database. i've removed references to the id, and instead use the org name to filter, in https://github.com/datamade/chi-councilmatic/pull/252/commits/c35675acbe4976546485dccf111bd9c0b1b7a3e5 and upstream in https://github.com/datamade/django-councilmatic/pull/240/commits/373a77bf086aba7cb2057901a7980482935b6fdc.